docs/topics/basket.rst

```diff
@@ -0,0 +1,149 @@
+Basket Synchronisation
+======================
+
+This documents what data we synchronize with `Basket <https://basket.readthedocs.io/>`_  and how.
+
+Triggers
+--------
+
+Every time a field that we're meant to synchronize on an object on changes, a full sync of the
```

Comment:
"to synchronize on an object on changes" - extra "on"?

Reply:
49ee7c029c2132559456a8be2af7240f49afa603